src/components/block/index.ts

```diff
@@ -910,7 +829,138 @@ export default class Block extends EventsDispatcher<BlockEvents> {
       input.removeEventListener('focus', this.handleFocus);
 
       if ($.isNativeInput(input)) {
-        input.removeEventListener('input', this.didMutated);
+        input.removeEventListener('input', this.didMutated as EventListener);
```

Comment:
You could try to use function overload for didMutated method to not cast here

Reply:
and also we'll need to cast our input to `BlockInputHTMLElement` anyway